app/code/OleksiiBezpoiasnyi/ControllerDemo/Controller/Git/Forward.php

```diff
@@ -0,0 +1,47 @@
+<?php
+declare(strict_types=1);
+
+namespace OleksiiBezpoiasnyi\ControllerDemo\Controller\Git;
+
+use Magento\Framework\Controller\Result\Forward as ForwardResponse;
+
+class Forward implements \Magento\Framework\App\Action\HttpGetActionInterface
+{
+    /**
+     * @var \Magento\Framework\App\RequestInterface $request
+     */
+    private $request;
+
+    /**
+     * @var \Magento\Framework\Controller\Result\ForwardFactory $forwardResponseFactory
+     */
+    private $forwardResponseFactory;
+
+    /**
+     * Controller constructor.
+     * @param \Magento\Framework\App\RequestInterface $request
+     * @param \Magento\Framework\Controller\Result\ForwardFactory $forwardResponseFactory
+     */
+    public function __construct(
+        \Magento\Framework\App\RequestInterface $request,
+        \Magento\Framework\Controller\Result\ForwardFactory $forwardResponseFactory
+    ) {
+        $this->request = $request;
+        $this->forwardResponseFactory = $forwardResponseFactory;
+    }
+    /**
+     * @return ForwardResponse
+     */
+    public function execute(): ForwardResponse
+    {
+        $result = $this->forwardResponseFactory->create();
+        return $result
```

Comment:
@AlexeyBezpoyasny The call of `setParams()` method should be at the same line as variable name.